crates/voxelia-client/src/graphics.rs

```diff
@@ -18,6 +18,8 @@ pub struct Graphics {
     pub pass: PhongPass,
     pub projection: Projection,
     pub camera: Camera,
+    pub camera_controller: CameraController,
+    pub mouse_pressed: bool,
```

Comment:
Move this mouse_pressed to the camera_controller, I think that's out of the scope of the graphics.